internal/internal_nexus_task_handler.go

```diff
@@ -217,11 +240,21 @@ func (h *nexusTaskHandler) handleStartOperation(ctx context.Context, nctx *Nexus
 	}
 	switch t := opres.(type) {
 	case *nexus.HandlerStartOperationResultAsync:
+		var links []*nexuspb.Link
+		for _, nexusLink := range t.Links {
+			links = append(links, &nexuspb.Link{
+				Url:  nexusLink.URL.String(),
```

Comment:
Hmm... I wonder if it would be worth escaping the raw query. It doesn't seem like `String()` does that for you. But this is probably fine, the server would just ignore if the URL can't be parsed.